R/vad.R

```diff
@@ -0,0 +1,183 @@
+globalVariables(c("DBZH","VRADH","RHOHV"))
+NULL
+
+#' Create a Velocity Azimuth Display (VAD) plot
+#'
+#' A velocity azimuth display plot visualized the radial velocity (`VRAD`) as a function of the azimuth from the radar.
+#' Among others it can be used to asses the fit of the movement speeds in a vertical profile.
+#' For example, when the movement is not uniform but rather in multiple directions or rotating this will be visible as
+#' deviations from the sine function.
+#'
+#' @param x A polar volume from which range gates are extracted.
+#' @param ... Currently not used.
+#' @param vp A vertical profile to annotate the VAD plot with the fit in the vertical profile
+#' @param range The distance range in to filter the range gates with.
+#'      If a `vp` is provided the range is taken from there and the argument `range` should not be provided.
+#' @param height The height range to filter the range gates by.
+#'      If only one value is provided next to a `vp` then the height bin intersection with this elevation is plotted.
+#' @param range_gate_filter Optional filtering of the range gates. By default range gates are filtered for a `DBZH` less then 20 and a `RHOHV` less then 0.95.
```

Comment:
please use reflectivity filtering on the linear eta scale (see dbz_to_eta()) - vol2bird currently uses 36000 as the cutoff. This has the advantage that effectively the same threshold is used at C and S-band

Reply:
or alternatively the default should look at whether the radar is S- or C-band and adjust the default threshold, 20 dBZ is too low for S-band